addons/packages/whereabouts/0.5.0/README.md

```diff
@@ -0,0 +1,108 @@
+# whereabouts Package
+
+This package provides the ability to assign IP addresses dynamically across your Kubernetes cluster using a CNI IPAM plugin named [whereabouts](https://github.com/k8snetworkplumbingwg/whereabouts).
+
+## Components
+
+* Whereabouts Custom Resources
+* Whereabouts DaemonSet
+* Whereabouts ServiceAccount
+* Whereabouts ClusterRoleBinding
+* Whereabouts ip-reconciler Cronjob
+
+## Configuration
+
+The following configuration values can be set to customize the whereabouts installation.
+
+### Global
+
+| Value | Required/Optional | Description |
+|-------|-------------------|-------------|
+| `namespace` | Optional | The namespace in which to deploy whereabouts components. Default: kube-system |
+
+### whereabouts Configuration
+
+| Value | Required/Optional | Description |
+|-------|-------------------|-------------|
+| `whereabouts.config.resources.limits.cpu` | Optional | The limits for CPU resources of whereabouts DeamonSet  |
+| `whereabouts.config.resources.limits.memory` | Optional | The limits for memory resources of whereabouts DeamonSet  |
+| `whereabouts.config.resources.requests.cpu` | Optional | The requests for CPU resources of whereabouts DeamonSet  |
+| `whereabouts.config.resources.requests.memory` | Optional | The requests for memory resources of whereabouts DeamonSet  |
+| `ip_reconciler.config.schedule` | Optional | The schedule of ip-reconciler CronJob. Default: \*/5 \* \* \* \*  |
+| `ip_reconciler.config.resources.requests.cpu` | Optional | The requests for memory resources of ip-reconciler CronJob  |
+| `ip_reconciler.config.resources.requests.memory` | Optional | The requests for memory resources of ip-reconciler CronJob  |
+
+## Usage
+
+The follow is a basic guide for getting started with whereabouts.
+
+This example guides you about attaching the second network interface to a pod with IP address assigned in the range you specified using whereabouts.
+
+1. Install TCE Multus CNI package to support multiple network by following
+   [doc](https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/multus-cni/3.7.1/README.md#usage-example):
+
+1. Install TCE whereabouts package through Tanzu CLI
+
+    ```bash
+    tanzu package install whereabouts --package-name whereabouts.community.tanzu.vmware.com --version ${MULTUS_PACKAGE_VERSION}
```

Comment:
Shoudl the version field be the whereabouts version?

Reply:
Good catch, will update it.